core/src/main/resources/AppConstants.properties

```diff
@@ -127,6 +127,19 @@ Statistics.size.boundaries=100000,1000000
 # Percentiles for StatisticsKeeper.
 Statistics.percentiles=50,90,95,98
 
+management.metrics.export.cloudwatch.namespace=
```

Comment:
misschien dat registry handiger is ipv export.

Reply:
Deze keys heb ik 1:1 overgenomen van Spring Boot